packages/js/src/plugins/auctionHouseModule/operations/executeSale.ts

```diff
@@ -308,11 +320,18 @@ export const executeSaleBuilder = (
   if (hasAuctioneer && !auctioneerAuthority) {
     throw new AuctioneerAuthorityRequiredError();
   }
+  if (isPartialSale && hasAuctioneer) {
+    throw new AuctioneerPartialSaleNotSupportedError();
+  }
+  if (isPartialSale) {
+    const buyerPricePerToken = buyerPrice.basisPoints.div(
+      buyerTokensSize.basisPoints
+    );
 
-  // Data.
-  const price = isNative
-    ? lamports(bid.price.basisPoints)
-    : amount(bid.price.basisPoints, treasuryMint.currency);
```

Comment:
Since we don't seem to be using prices as amounts anymore, is there a need to require the `treasuryMint` property of the `auctionHouse` parameter anymore? It might be used somewhere else but I just want to make sure we're not requiring more properties than we need to keep the "Minimum Valuable Input" principle.

Reply:
Yes, it's still needed to provide treasuryMint address to the instruction and also we need a full Auction House model to build a LazyPurchase model instance.

Reply:
Make sense, thanks for clarifying!